runtime-config-linux.md

```diff
@@ -17,7 +17,7 @@ The following parameters can be specified to setup namespaces:
     * **`uts`** the container will be able to have its own hostname and domain name
     * **`user`** the container will be able to remap user and group IDs from the host to local users and groups within the container
 
-* **`path`** *(string, optional)* - path to namespace file
+* **`path`** *(string, optional)* - path to namespace file in the host mount namespace
```

Comment:
On Tue, Dec 22, 2015 at 10:32:46PM -0800, Brandon Philips wrote:
> > -* **`path`** *(string, optional)* - path to namespace file
> > +* **`path`** *(string, optional)* - path to namespace file in the host mount namespace
> 
> I think "runtime namespace" is the most compact way of expressing
> this. We can expand that meaning in a glossary.

Ok, that makes two votes in favor of “runtime” phrasing [1] after I'd
written up my reasons for preferring host in detail [2].  I'll rebase
this PR to use that language and add it to the glossary once something
like #107 lands and we have a glossary.

[1]: https://github.com/opencontainers/specs/pull/275/files#r48057211
[2]: https://github.com/opencontainers/specs/pull/275/files#r48059428